Tasks/Common/TlsHelper_/TlsHelper_.psm1

```diff
@@ -0,0 +1,12 @@
+function Add-Tls12InSession {
+    [CmdletBinding()]
+    param()
+
+    try {
+        [Net.ServicePointManager]::SecurityProtocol += [Net.SecurityProtocolType]::Tls12
```

Comment:
first check if TLS1.2 is already in protocol.

Reply:
Checking now